### PR TITLE
Add example script on how to update contract products

### PR DESCRIPTION
**Pull request by eliakaris**
This change adds a new script, `update-product-tags.py` which gives an example for editing contract products.

This allows adding product tags and custom fields.

Details are in the readme

### CODE REVIEW

File: scripts/update-product-tags/update-product-tags.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+import argparse
+import os
+import csv
+import random
+import uuid
+import requests
+from dotenv import load_dotenv
+load_dotenv()
+
+load_dotenv()  # take environment variables from .env.
```
Comment (keyworks) [inline scripts/update-product-tags/update-product-tags.py:13]: double load_dotenv()?
```suggestion

load_dotenv()  # take environment variables from .env.
```

File: scripts/update-product-tags/update-product-tags.py [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    raise ValueError("Set API_TOKEN environment variable to metronome API key")
+
+args = parser.parse_args()
+csv_file_path = args.csv_file
+if not csv_file_path:
+    raise ValueError("csv_file must be set")
+
+api_url = os.getenv('API_URL')
+if not api_url:
+    api_url = 'https://api.metronome.com'
```
Comment (keyworks) [inline scripts/update-product-tags/update-product-tags.py:29]:
```suggestion
api_url = os.getenv('API_URL', 'https://api.metronome.com')
```

File: scripts/update-product-tags/update-product-tags.py [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+
+    # check for custom field columns, create if not found
+    custom_field_columns = []
+    for column in csv_reader.fieldnames:
+        if column.startswith('field:'):
+            print(f"Custom field column found: {column}")
+            custom_field = column.split(':')[1]
+            custom_field_columns.append(custom_field)
+
+            if not custom_field in current_custom_fields:
```
Comment (keyworks) [inline scripts/update-product-tags/update-product-tags.py:95]:
```suggestion
            if custom_field not in current_custom_fields:
```

**Review by keyworks — DISMISSED** [review]

File: scripts/update-product-tags/update-product-tags.py (hunk 1 above)
Comment (eliakaris) [reply scripts/update-product-tags/update-product-tags.py:13]: i really want to load the env

**Review by keyworks — APPROVED** [review]

**Pull request merged**